modules/publicize/ui.php

```diff
@@ -18,6 +18,10 @@ function __construct() {
 
 		$this->publicize = $publicize = new Publicize;
 
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	function init() {
```

Comment:
What do you say we move the `current_user_can` check up to avoid setting the action hook if the user has no privileges?